gradle.properties

```diff
@@ -15,21 +15,19 @@
 #
 # project metadata used for publications
 group=io.servicetalk
-version=0.42.0
+version=0.42.11
 scmHost=github.com
 scmPath=servicetalk/examples
 issueManagementUrl=https://github.com/servicetalk/examples/issues
 ciManagementUrl=https://github.com/servicetalk/examples/actions
 
 # dependency versions
-servicetalkVersion=0.42.10
-log4jVersion=2.17.2
+servicetalkVersion=0.42.11
 slf4jVersion=1.7.36
 
 jaxRsVersion=2.1.6
 jerseyVersion=2.35
 
-jacksonVersion=2.13.2.2
```

Comment:
Why were only some versions removed from this file, but not most of them?

Reply:
Initially it was because I wanted to be able to switch between `servicetalk-bom` and `servicetalk-dependencies` for testing the "how to override version" example but I never thought to remove the versions after completing testing.